templates/jvb-statefulset.yaml

```diff
@@ -66,6 +66,9 @@ spec:
           requests:
             cpu: 100m
             memory: 100Mi
+        {{- with $.Values.jvb.monitoring.containerExtraSpec }}
```

Comment:
Either this one or the below is likely incorrect as these should be at the same level?

Reply:
Indeed there was a mistake, I missed the monitoring spec on my second pass.